media-sound/schismtracker/schismtracker-20120105.recipe

```diff
@@ -49,9 +49,11 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	aclocal
+	autoreconf -vfi
 	runConfigure ./configure
 	make $jobArgs
+
+	addAppDeskbarSymlink "$binDir"/schismtracker SchismTracker
```

Comment:
addAppDeskbarSymlink usually goes in INSTALL().

Reply:
jikes ;)